package.json

```diff
@@ -77,6 +77,7 @@
     "grunt-contrib-copy": "1.0.0",
     "grunt-shell": "3.0.1",
     "grunt-wp-deploy": "2.1.2",
+    "ignore-emit-webpack-plugin": "2.0.6",
```

Comment:
@pierlon Because of this addition and the fact that I'm on npm v7, the `lockfileVersion` in the `package-lock.json` file has been bumped to `2`.

According to the [docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion), `lockfileVersion: 2` is backward-compatible to v1 lockfiles so I guess we're safe with that.

Can you confirm this is all good?

Reply:
This is related to the discussion on https://github.com/ampproject/amp-wp/pull/6001#issuecomment-810064829

Reply:
> Can you confirm this is all good?

Yep, the CI is also using npm v7 so we shouldn't be facing any issues regarding `lockfileVersion`.